src/components/Router/useOptionsSteps.tsx

```diff
@@ -0,0 +1,25 @@
+import { useCallback, useEffect, useState } from 'preact/hooks'
+import { StepsHook, StepsLoadingStatus } from '~types/routers'
+import { StepConfig } from '~types/steps'
+import useUserConsent from '~contexts/useUserConsent'
+import { NarrowSdkOptions } from '~types/commons'
+
+export const createOptionsSteps = (
```

Comment:
Small suggestion: Rename to `createOptionsStepsHook` to indicated better what it does without needing to read to code. Also rename file to the same.